lib/parser.js

```diff
@@ -116,6 +117,144 @@ function isDogescriptSource(keys)
   return false;
 }
 
+function handleWoof(keys)
+{
+
+  // woof foo is X
+  if ( keys[2] === 'is')
```

Comment:
Is the potential ambiguity between this and exporting an equality comparison a concern?

Reply:
I battled with this internally. originally I was using the `as` keyword like we do for require: `so blah as blah`, but that sort of shifted everything around:

```
woof such f much b wow as bar
```

Can you export an `===` ? :
```
module.exports = === b;
```

I kept `is` since at least it feels consistent (though misleading) with `very x is y`. I guess it's not really needed for the `woof` statement. Would it feel too out of place to not have it?

```
woof bar such foo
```

vs
```
woof bar is such foo
```

Reply:
Or maybe I misunderstood :( care to elaborate? @vpzomtrrfrt

Reply:
I think I was referring to the possibility of `module.exports = foo === bar`

Reply:
I probably wouldn't expect it to work in 2.x, but how would that export be written in djs?

`woof foo is bar` seems like it could be either `module.exports.foo = bar` or `module.exports = foo === bar`

Reply:
Ah... yeah I didn’t consider that. It would do `module.exports.foo = bar`

I guess if you really needed module.exports = foo === bar you could just write it in JS since we ignore non dogescript source.

Reply:
Well, drat....

would `as` instead of `is` be better? It sort of goes with the `so blah as blah2`

Reply:
@Pholey thoughts?

Reply:
So, my personal preference is to keep `is` despite ambiguity (since the usage of `so` sort of flips things around). 

@vpzomtrrfrt  or @Pholey  (any preference for one over another). I don't see an immediate use for a:

```
module.exports. = foo === bar
```

So not supporting that doesn't seem like a big deal to me

Reply:
While this is likely a rare construct, I'd like to eventually be able to write any non-esoteric Javascript code in native Dogescript. `as` seems like a good solution, especially with the similarity to `so`'s grammar

Reply:
Ambiguity causes confusion, and seems like a good thing to avoid

Reply:
We might have to make it a bit wonky:

```
woof as foo much a b
  shh 1
wow
```

Since wow is a terminating keyword it might be tricky to try to support

```
woof much a b
ahh 1
wow as foo
```

Particularly with the available tokens being provided on a line basis, we won’t have the export name until we process the `wow`

Reply:
We might have to make it a bit wonky:

```
woof as foo much a b
  shh 1
wow
```

Since wow is a terminating keyword it might be tricky to try to support

```
woof much a b
ahh 1
wow as foo
```

Particularly with the available tokens being provided on a line basis, we won’t have the export name until we process the `wow`

Reply:
Hmm, that's a good point

Reply:
What if we used `so` (feels odd but the woof keyword helps distinguish)

```
woof foo so much a b
  shh 1
wow
```

```
module.exports.foo = function(a,b,) { //1 }
```

Reply:
Well, other than`so` I also thought of `be`

`woof foo be much a b wow` => module.exports.foo = function (a,b)

Can’t think of any other keywords atm :(